docs/data/sdks/android/ampli.md

```diff
@@ -3,16 +3,16 @@ title: Android Ampli Wrapper
 description: Learn how to install and use the Amplitude Data Ampli Wrapper for the Android Java and Kotlin runtimes.
 ---
 
-!!!note
-    This page covers the Android Java and Kotlin runtimes. All (Itly) runtimes are deprecated. If you are still using an (Itly) runtime, see the **[migration guide](#migrating-from-previous-version)** to upgrade to the newest runtime. Docs for the Itly version are available **[here](browser)**.
 
 Amplitude Data supports tracking analytics events from Android apps written in Kotlin and Java.
 
 In Kotlin and Java, the tracking library exposes a type-safe function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for code completion and compile-time checks.
 
-!!!tip
+!!!example "Ampli Example"
```

Comment:
To keep this similar to the core SDK docs I was thinking we could display this like `SDK Resources`. WDYT? cc @caseyamp 

```
Ampli Resources
* Android Java Ampli Examples
* Android Kotlin Ampli Examples
```

<img width="711" alt="image" src="https://user-images.githubusercontent.com/5790872/186750452-ea0441c7-7a0a-4582-8f95-da157533f62a.png">